editor/assets/default_renderpipeline/builtin-pipeline.ts

```diff
@@ -438,7 +438,7 @@ if (rendering) {
                 : nativeHeight;
 
             // Render Window (UI)
-            ppl.addRenderWindow(window.colorName, Format.BGRA8, nativeWidth, nativeHeight, window);
+            ppl.addRenderWindow(window.colorName, Format.RGBA8, nativeWidth, nativeHeight, window);
```

Comment:
Here I have made a design mistake. We shouldn't specify texture format in `addRenderWindow`. I suggest we don't use the 'format' parameter and get the format from the render window directly.
The reason is that the swapchain format might be 'rgba8' or 'bgra8', which is platform dependent. So we need to get the format from `RenderWindow` instead of hardcode.